db/seeds.rb

```diff
@@ -5,5 +5,5 @@
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
-puts "Importing Schools information, this may take a minute..."
-SchoolDataImporter.new.run
+
+FactoryBot.create(:school, :student_loan_eligible, :maths_and_physics_eligible)
```

Comment:
Having just had a pretty pleasant onboarding experience, I can say I think it's good for a new developer to be able to play around with the app out of the box (i.e. after seeding the database) and experience all the main real-world scenarios. For this app, that includes selecting an ineligible school. So it might be worth adding a couple of ineligible schools here, e.g. a closed one and one with an ineligible local authority.

Reply:
Yeah I'll take a look at this, it will be nice to be able to reuse the ones in `spec/fixtures`